src/ucp/core/ucp_ep.h

```diff
@@ -173,6 +174,11 @@ typedef struct ucp_ep_config {
         } offload;
     } tag;
 
+    struct {
+        /* Protocols used for stream operations
+         * (currently it's only AM based). */
+        const ucp_proto_t   *proto;
```

Comment:
why do we need this now if there is only one protocol?
also, for tags, config structure does not contain pointer to proto structure..

Reply:
I'll add at least bcopy and zcopy in next PRs. config structure contains pointer to proto structure - see config.tag->proto

Reply:
right, missed it.